diff --git a/target/linux/octeon/config-4.14 b/target/linux/octeon/config-4.14
index 24cba2b80a..d1b6ffab59 100644
--- a/target/linux/octeon/config-4.14
+++ b/target/linux/octeon/config-4.14
@@ -319,3 +319,13 @@ CONFIG_WEAK_ORDERING=y
 CONFIG_XPS=y
 CONFIG_ZLIB_INFLATE=y
 CONFIG_ZONE_DMA32=y
+CONFIG_NF_FLOW_TABLE=y
+CONFIG_NF_FLOW_TABLE_HW=y
+CONFIG_NFT_FLOW_OFFLOAD=y
+CONFIG_NF_FLOW_TABLE_INET=y
+CONFIG_NF_FLOW_TABLE_IPV4=y
+CONFIG_NF_FLOW_TABLE_IPV6=y
+CONFIG_TCP_CONG_BBR=y
+CONFIG_NET_SCH_FQ=y
+CONFIG_TCP_CONG_ADVANCED=y
+CONFIG_DEFAULT_BBR=y
\ No newline at end of file
